site/en/docs/crux/history-api/index.md

```diff
@@ -0,0 +1,617 @@
+---
```

Comment:
question: Also, from what I understand, this is relatively similar functionality to current CrUX API so maybe it would be more beneficial to keep it short and up to the point on how does it correlate with the current CrUX API?

Please, keep in mind that I might be missing some parts here. If so, I will try to correct myself :)

Reply:
You're totally right. I struggled between the different options of:
- Adding to that existing api doc
- Keeping this very short
- Copying that doc and ruthlessly cutting as much as I could out of it, but still leaving it long enough to sit as a semi-standalone reference doc.

I went with the third option, but would value your opinion on better ways to do this!

Reply:
I am a huge fan of short and up-to-the-point documentation or articles. 

If this new History API is quite similar to the existing API, I would focus in this document on explaining the core/major differences and for everything that is common, just add a link to the existing API documentation so that it would be easier for the new users to start this out.

So preferably option 2 ;)

1. What is the new History API
2. Use Cases
3. Major differences between History API and existing API
4. Links to the existing API docs

This is probably how I would have done that :)

Reply:
That sounds more like the blog post we're working on! As I say I'd like to keep the "docs" more a reference. Let's pick this up again after I'd added the blog post to this PR and see what you think.

Reply:
Sounds great. Let me know once it is added and I will take a look at it :)

Reply:
@Baroshem the blog post has been added to this PR if you get a chance to look:
https://deploy-preview-5087--developer-chrome-com.netlify.app/blog/chrome-ux-report-history-api/

Hoping to release this later today.